SendUserDataProcessingEmailActivity/index.ts

```diff
@@ -63,10 +86,16 @@ const mailerTransporter = isProduction
       secure: false
     });
 
+const sendMailTask = (mt: Mail) => (
+  options: Mail.Options & { html: Mail.Options["html"] }
```

Comment:
while the proposed changes fix the error, they don't prevent the error (missing html param) to occur again in the future so I've added a required html parameter here